src/client/conn/http1.rs

```diff
@@ -402,6 +402,19 @@ impl Builder {
         self
     }
 
+    /// Set whether HTTP/1 connections will silently ignored malformed header lines.
+    ///
+    /// If this is enabled and a header line does not start with a valid header
+    /// name, or does not include a colon at all, the line will be silently ignored
+    /// and no error will be reported.
+    ///
+    /// Default is false.
+    pub fn ignore_invalid_headers_in_requests(&mut self, enabled: bool) -> &mut Builder {
```

Comment:
This should be in the `server` module. We likely can also drop off the `_in_requests` suffix, since it's the only message that would make sense for a server (the responses cannot be constructed with invalid headers at all, before passing to the connection).

Reply:
would be same for `ignore_invalid_headers_in_responses` though, but I guess you cannot drop that for backwards compatibility